core/src/main/java/org/mobilitydata/gtfsvalidator/table/AnyTableLoader.java

```diff
@@ -32,10 +32,14 @@ public final class AnyTableLoader {
   private static final FluentLogger logger = FluentLogger.forEnclosingClass();
   private static final List<Class<? extends FileValidator>> singleFileValidatorsWithParsingErrors =
       new ArrayList<>();
-
+  private static boolean hasTranslations = false;
```

Comment:
This field is not necessary, as the presence of the translation file can be detected by the method [GtfsTableContainer.isMissing](https://github.com/MobilityData/gtfs-validator/blob/b597b7c2521bae18041e2f7c9879304c33c16aa5/core/src/main/java/org/mobilitydata/gtfsvalidator/table/GtfsTableContainer.java#L103)